rest/main.go

```diff
@@ -60,9 +55,14 @@ func serverMain(ctx context.Context, osArgs []string) error {
 		return err
 	}
 
-	err = fillConfigWithFlags(fs, configFlags)
+	err := fillConfigWithFlags(fs, configFlags)
 	if err != nil {
-		return err
+		return fmt.Errorf("error merging flags on to config: %w", err)
+	}
+
+	err = fillConfigWithLegacyFlags(legacyConfigFlags, fs, flagStartupConfig.Logging.Console.LogLevel != nil)
```

Comment:
I think filling legacy flags AFTER the normal flags will make them override non-legacy right?

Reply:
Yes, I can do that. The reason I made it not do that is so that it can check if `Logging.Console.LogLevel` has been set by the user, so then a warning can be issued if `verbose` is set.

Will swap the order, and use mergo to make sure persistent config flags take priority.